tools/notifyServicelogEvent.c

```diff
@@ -23,8 +23,12 @@
 #include <inttypes.h>  // PRIu64, SCNu64
 
 #include <crm/common/attrd_internal.h>
+#include <crm/common/cmdline_internal.h>
 #include <crm/common/ipc_attrd_internal.h>
 
+const char *summary = "notifyServicelogEvent - handle events written to "
+                      "servicelog database";
+
```

Comment:
There's no good reason for it, but every other tool is using `#define SUMMARY ...` (or at least, every tool I've converted) and I'm a big fan of things looking the same.

Speaking of consistency, we are not at all consistent about including the program name in the description.  Some tools do, some tools don't.  I think it looks better when they do, so we need to go through the others and make that change.

Reply:
My bad, I was looking at `crm_verify.c` when I did that. https://github.com/ClusterLabs/pacemaker/blob/main/tools/crm_verify.c#L33-L38